packages/app/package.json

```diff
@@ -6,14 +6,16 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "nook:watch": "bun --bun nook-agent"
   },
   "dependencies": {
     "next": "14.0.0",
+    "nook": "0.0.1",
+    "nook-agent": "0.0.1",
```

Comment:
I assume as a person using the package, I would want to only install one package providing me with everything required. So probably at this point I would think about how exactly are we going to integrate with the frameworks. We can start with a single framework like Next.js in case they depend on different build systems. 

For this use case, I think that if we want to start a server ourselves – it should still happen without running an additional command but have it included in the dev server process. Ideally anything we do should be invisible to the product developers, e.g. they wrap their components with nook and they keep developing the product as usual without having to install anything their product might not have out of the box